CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Changelog
 
+## v1.5.0
+(Just for temporary reference, this is the last commit considered: 71566ae832ae120037a31591ba0b72305a6f3b2a)
+
+In this new release of AiiDA the support for python 3.5 is dropped [[#4386]](https://github.com/aiidateam/aiida-core/pull/4386).
+This version remains compatible with all current python versions that are not end-of-life:
+ * 3.6
+ * 3.7
+ * 3.8
+
+### Features
+- REST API: list endpoints at base URL [[#4412]](https://github.com/aiidateam/aiida-core/pull/4412)
+- `ProcessBuilder`: allow unsetting of inputs through attribute deletion [[#4419]](https://github.com/aiidateam/aiida-core/pull/4419)
+- `verdi migrate`: make `--in-place` work across different file systems [[#4393]](https://github.com/aiidateam/aiida-core/pull/4393)
+
+### Improvements
+- Refactor Import Archive [[#4510]](https://github.com/aiidateam/aiida-core/pull/4510)
+- The remaining old content has been incorporated in the new documentation
+- Archive exporter: refactor progress bar [[#4504]](https://github.com/aiidateam/aiida-core/pull/4504)
+- Archive exporter: decouple the extraction of data from the writing to file [[#4448]](https://github.com/aiidateam/aiida-core/pull/4448)
+- `verdi process show`: order by ctime and print process label [[#4407]](https://github.com/aiidateam/aiida-core/pull/4407)
+- `LinkManager`: fix inaccuracy in exception message for non-existent link [[#4388]](https://github.com/aiidateam/aiida-core/pull/4388)
+
+### Bug fixes
+- Fix `ZeroDivisionError` in worker slots check [[#4513]](https://github.com/aiidateam/aiida-core/pull/4513)
+- Fix `UnboundLocalError` in `aiida.cmdline.utils.edit_multiline_template` [[#4436]](https://github.com/aiidateam/aiida-core/pull/4436)
+- Only remove temporary folder if it is present [[#4379]](https://github.com/aiidateam/aiida-core/pull/4379)
+- Improve the deprecation warning for `Node.open` outside context manager [[#4434]](https://github.com/aiidateam/aiida-core/pull/4434)
```

Comment:
Would make this an improvement

Reply:
Just to make sure: I see selected all of the entries of bug fixes, you meant all of them? In that case I will directly eliminate the category.

Reply:
I was talking just about "Improve the deprecation....". We should keep the bug fix category